fsthttp/limits.go

```diff
@@ -0,0 +1,49 @@
+package fsthttp
+
+import "github.com/fastly/compute-sdk-go/internal/abi/fastly"
+
+// Limits handles HTTP limits
+var Limits = limits{}
+
+type limits struct {
+}
+
+// GetMaxHeaderNameLen gets the header name limit
+func (limits) GetMaxHeaderNameLen() int {
```

Comment:
Effective Go recommends not having getter function names start with `Get`.  This should jus be `MaxHeaderNameLen()`.

Reply:
Yes, I agree, it reads better. :)